pkg/valueset/postgresql_driver.go

```diff
@@ -0,0 +1,58 @@
+package valueset
+
+import (
+	"fmt"
+
+	"github.com/jackc/pgx/v5"
+	_ "github.com/jackc/pgx/v5/stdlib"
+)
+
+// match (n:OMOPConcept) return n
+func GetResults(cfg Config, queryParams map[string]string) (interface{}, error) {
+	ret := make([]map[string]interface{}, 0)
+DATABASE:
+	for _, dbs := range cfg.Databases {
+		for _, vs := range dbs.Valuesets {
+			if err := cfg.GetConnection(vs, &dbs.Database); err != nil {
+				return nil, err
+			}
+			if dbs.Driver == nil {
+				continue DATABASE
+			}
+			for _, query := range vs.Queries {
+				for _, val := range queryParams {
+					rows, err := dbs.Driver.Query(query.Query, pgx.NamedArgs{"concept_id": val})
+					if err != nil {
+						return nil, err
+					}
+					cols, err := rows.Columns()
+					if err != nil {
+						return nil, err
+					}
+
+					for rows.Next() {
```

Comment:
If 0 rows, move on to the next query for the tableId
If 1 row, that's the result
if >1 rows, error